po/fr.po

```diff
@@ -0,0 +1,296 @@
+# French translation for Ladybird.
+msgid ""
+msgstr ""
+"Project-Id-Version: ladybird wip\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-09-24 23:32+0300\n"
+"Language: fr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+
+#. Translators: this is a format string for the zoom-percent label in the main menu.
+#. For most languages, it doesn't need translating.
+#: Window.cpp:539
+#, c-format
+msgid "%d%%"
+msgstr ""
+
+#: Window.cpp:445
+msgid "A script closed the web page"
+msgstr "Un script a fermé la page web"
+
+#: window.blp:205 window.blp:306
+msgid "Bookmarks"
+msgstr "Marque-pages"
+
+#: window.blp:51
+msgid "Bookmarks & History"
+msgstr "Marque-pages & Historique"
+
+#: Tab.cpp:215 Tab.cpp:239
+msgid "Cancel"
+msgstr "Annuler"
+
+#: window.blp:37
+msgid "Close _Other Tabs"
+msgstr "Fermer _les autres onglets"
+
+#: shortcuts-dialog.blp:63
+msgid "Close current tab"
+msgstr "Fermer l‘onglet actif"
+
+#: Tab.cpp:216
+msgid "Confirm"
+msgstr "Confirmer"
+
+#: shortcuts-dialog.blp:97 shortcuts-dialog.blp:99
+#, fuzzy
+msgid "Developer Tools"
+msgstr "Outils de développement"
+
+#: Tab.cpp:240
+msgid "Enter"
+msgstr "OK"
+
+#: js-console.blp:12
+msgid "Enter a JavaScript expression to evaluate"
+msgstr "Entrer l‘expression JavaScript à évaluer"
+
+#: Window.cpp:234
+msgid "Failed to pick file"
+msgstr "Impossible d‘ouvrir le fichier"
+
+#: shortcuts-dialog.blp:50
+msgid "Focus location entry"
+msgstr "Donner le focus à la barre d‘adresse"
+
+#: shortcuts-dialog.blp:10
+msgid "General"
+msgstr "Général"
+
+#: Tab.cpp:48 shortcuts-dialog.blp:102 window.blp:16
+msgid "JavaScript Console"
+msgstr "Console JavaScript"
+
+#: Application.cpp:66
+msgid ""
+"Ladybird is a browser based on LibWeb web engine and LibJS JavaScript "
+"engine, developed by a large team of contributors as a part of the "
+"SerenityOS project."
+msgstr ""
+"Ladybird est un navigateur basé sur le moteur de rendu LibWeb et la bibliothèque JavaScript "
+"LibJS, développé par une équipe de contributeurs appartenant au projet SerenityOS."
+
+#: window.blp:218
+msgid "Main Menu"
+msgstr "Menu Principal"
+
+#: window.blp:145 window.blp:282
+msgid "Navigate Back"
+msgstr "Naviguer vers la page précédente"
+
+#: window.blp:157 window.blp:288
+msgid "Navigate Forward"
+msgstr "Naviguer vers la page suivante"
+
+#: shortcuts-dialog.blp:38
+msgid "Navigate back"
+msgstr "Page précédente"
+
+#: shortcuts-dialog.blp:42
+msgid "Navigate forward"
+msgstr "Page suivante"
+
+#: shortcuts-dialog.blp:35
+msgid "Navigation"
+msgstr "Navigation"
+
+#: window.blp:175
+msgid "New Tab"
+msgstr "Nouvel Onglet"
+
+#: window.blp:11 window.blp:43
+msgid "New _Incognito Window"
+msgstr "Nouvelle Fenêtre de Navigation _Privée"
```

Comment:
I have a slight preference for "en Navigation Privée". It's only a suggestion tho
(If you change it, there is a second occurrence that I can't find on my phone)

Reply:
(please also see my comment on terminology above)